<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>773</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>- SimpleCAM - </string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/application-system.png</normaloff>:/icons/application-system.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0,0">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0" colspan="2">
     <widget class="QWidget" name="main" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QFrame" name="info">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="2">
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1,0,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="xMin">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>489,57</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string> / </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="yMin">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>489,57</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string> / </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="zMin">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>489,57</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QFrame" name="frame_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="message">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QFrame" name="frame_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="xMax">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="lineWidth">
             <number>-5</number>
            </property>
            <property name="text">
             <string>489,57</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string> / </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="yMax">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>489,57</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string> / </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="zMax">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>489,57</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFrame" name="frame_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="material">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Wasser</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFileOpen"/>
    <addaction name="actionProjectOpen"/>
    <addaction name="actionToolsOpen"/>
    <addaction name="separator"/>
    <addaction name="actionProjectSave"/>
    <addaction name="actionToolsSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionOrthografic"/>
    <addaction name="separator"/>
    <addaction name="actionIso1"/>
    <addaction name="actionIso2"/>
    <addaction name="actionIso3"/>
    <addaction name="actionIso4"/>
    <addaction name="separator"/>
    <addaction name="actionTop"/>
    <addaction name="actionBottom"/>
    <addaction name="actionLeft"/>
    <addaction name="actionRight"/>
    <addaction name="actionFront"/>
    <addaction name="actionBack"/>
    <addaction name="separator"/>
    <addaction name="actionZoom2Fit"/>
    <addaction name="separator"/>
    <addaction name="actionHideModel"/>
    <addaction name="actionWireframe"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOperations">
    <property name="title">
     <string>Operations</string>
    </property>
    <widget class="QMenu" name="menu_New">
     <property name="title">
      <string>&amp;New</string>
     </property>
     <addaction name="actionContourNew"/>
     <addaction name="actionDrillNew"/>
     <addaction name="actionSweepNew"/>
    </widget>
    <addaction name="menu_New"/>
    <addaction name="actionToolPath"/>
    <addaction name="actionGenerate_GCode"/>
    <addaction name="separator"/>
    <addaction name="actionRemove"/>
   </widget>
   <widget class="QMenu" name="menuSetup">
    <property name="title">
     <string>Setup</string>
    </property>
    <addaction name="actionModelSetup"/>
    <addaction name="actionOperationsSetup"/>
    <addaction name="actionToolsSetup"/>
    <addaction name="actionMiscSetup"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOperations"/>
   <addaction name="menuView"/>
   <addaction name="menuSetup"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionFileOpen">
   <property name="text">
    <string>open &amp;CAD File</string>
   </property>
  </action>
  <action name="actionOrthografic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Orthografic</string>
   </property>
  </action>
  <action name="actionTop">
   <property name="text">
    <string>&amp;Top</string>
   </property>
  </action>
  <action name="actionFront">
   <property name="text">
    <string>&amp;Front</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="text">
    <string>&amp;Left</string>
   </property>
  </action>
  <action name="actionBottom">
   <property name="text">
    <string>&amp;Bottom</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="text">
    <string>Back</string>
   </property>
   <property name="iconText">
    <string>B&amp;ack</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="text">
    <string>&amp;Right</string>
   </property>
  </action>
  <action name="actionZoom2Fit">
   <property name="text">
    <string>Zoom &amp;fit to Window</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionIso1">
   <property name="text">
    <string>Iso&amp;1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionIso2">
   <property name="text">
    <string>Iso&amp;2</string>
   </property>
  </action>
  <action name="actionProjectOpen">
   <property name="text">
    <string>open &amp;Project</string>
   </property>
  </action>
  <action name="actionProjectSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>sa&amp;ve Project as</string>
   </property>
   <property name="iconText">
    <string>save Project as</string>
   </property>
  </action>
  <action name="actionModelSetup">
   <property name="text">
    <string>Model</string>
   </property>
  </action>
  <action name="actionOperationsSetup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Operations</string>
   </property>
  </action>
  <action name="actionToolsSetup">
   <property name="text">
    <string>Tools</string>
   </property>
  </action>
  <action name="actionToolsOpen">
   <property name="text">
    <string>open &amp;Tools</string>
   </property>
  </action>
  <action name="actionToolsSave">
   <property name="text">
    <string>&amp;save Tools</string>
   </property>
  </action>
  <action name="actionHideModel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>hide Model</string>
   </property>
   <property name="iconText">
    <string>&amp;hide Model</string>
   </property>
  </action>
  <action name="actionSetupInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wireframe</string>
   </property>
  </action>
  <action name="actionToolPath">
   <property name="text">
    <string>&amp;Create Toolpath</string>
   </property>
  </action>
  <action name="actionGenerate_GCode">
   <property name="text">
    <string>&amp;Generate GCode</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionContourNew">
   <property name="text">
    <string>Contou&amp;r</string>
   </property>
  </action>
  <action name="actionDrillNew">
   <property name="text">
    <string>&amp;Drill Cycle</string>
   </property>
  </action>
  <action name="actionSweepNew">
   <property name="text">
    <string>S&amp;weep</string>
   </property>
  </action>
  <action name="actionMiscSetup">
   <property name="text">
    <string>Misc</string>
   </property>
  </action>
  <action name="actionIso3">
   <property name="text">
    <string>Iso&amp;3</string>
   </property>
  </action>
  <action name="actionIso4">
   <property name="text">
    <string>Iso&amp;4</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
